extensions/sql-database-projects/src/models/dataSources/dataSources.ts

```diff
@@ -53,11 +52,11 @@ export async function load(dataSourcesFilePath: string): Promise<DataSource[]> {
 	// TODO: do we have a construct for parsing version numbers?
 	switch (rawJsonContents.version) {
 		case '0.0.0':
-			const dataSources: DataSourceFileJson = rawJsonContents as DataSourceFileJson;
```

Comment:
can this be removed?

Reply:
@Benjin is it ok to remove this data sources code or should I just leave it commented out? Is the plan for data sources to be in the sql database projects extension or in the data workspaces extension in the future?